MediaBrowser.Common/Net/IPNetAddress.cs

```diff
@@ -123,15 +123,27 @@ public static bool TryParse(string addr, out IPNetAddress ip)
                     if (IPAddress.TryParse(tokens[0], out res))
                     {
                         // Is the subnet part a cidr?
-                        if (byte.TryParse(tokens[1], out byte cidr))
+                        if (int.TryParse(tokens[1], out int cidr))
```

Comment:
Why use int over byte? Any value above 255 is invalid

Reply:
cause 65535 would fail, and yet still be accepted by the IPAddress.TryParse as valid.

Reply:
👍🏻 Didn't think of that

Reply:
neither did I until it went wrong!